TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -464,6 +464,9 @@ TYPO3:
       'TYPO3.Neos:Preview':
         maximumWidth: 1000
         maximumHeight: 1000
+      'TYPO3.Neos:InspectorImage':
+        maximumWidth: 300
+        maximumHeight: 300
```

Comment:
we shouldn't include more presets as they multiply for each asset, thus if you have 1000 assets, you will get 3000 thumbnails instead of 2000 when using automatic thumbnail creation.. thus stick to the existing ones, either the 250 one suffice or else use the larger one